backend/tests/test_profile.py

```
@@ -0,0 +1,167 @@
+import sys
+from pathlib import Path
+import pytest
+from fastapi.testclient import TestClient
+from sqlalchemy import create_engine
```

Ruff (0.8.2)

`sqlalchemy.create_engine` imported but unused

Remove unused import: `sqlalchemy.create_engine`

(F401)